test/unit/tab/index.test.tsx

```diff
@@ -65,6 +65,44 @@ test('if props.active updates to false, foundation.deactivate is called', () =>
   td.verify(wrapper.instance().deactivate(), {times: 1});
 });
 
+test('calls foundation.setFocusOnActivate when props.focusOnActivate changes from false to true', () => {
+  const wrapper = shallow<Tab>(<Tab focusOnActivate={false} />);
+  wrapper.instance().foundation.setFocusOnActivate = td.func();
+  wrapper.setProps({focusOnActivate: true});
+  td.verify(wrapper.instance().foundation.setFocusOnActivate(true), {times: 1});
+});
+
+test('calls foundation.setFocusOnActivate when props.focusOnActivate changes from true to false', () => {
+  const wrapper = shallow<Tab>(<Tab focusOnActivate />);
+  wrapper.instance().foundation.setFocusOnActivate = td.func();
+  wrapper.setProps({focusOnActivate: false});
+  td.verify(wrapper.instance().foundation.setFocusOnActivate(false), {times: 1});
+});
+
+test('when props.focusOnActivate is true, an active tab should be focused on mount', () => {
+  const wrapper = mount<Tab>(<Tab active focusOnActivate />, {attachTo: document.body});
```

Comment:
just tried running tests using `npm run test:watch`. It seems like this isn't valid. Please try using this pattern: https://github.com/material-components/material-components-web-react/commit/764ed4017d716e8124614ff47bcac0064feca1ec#diff-5e69e52eb0131589a6b4841a086ff648R65

Reply:
Do you mean the warnings from react about rendering directly to the body? Seems I didn't see those when running locally 😢

Reply:
Yup! Those warnings -- The tests pass, but they do produce warnings. I saw you pushed the changes. I will check them out now